use_case_examples/lora_finetuning/README.md

```diff
@@ -56,6 +56,29 @@ Each notebook includes its own data-set:
 
 Execute the Jupyter notebook `GPT2FineTuneHybrid.ipynb` to start the fine-tuning process. The notebook is structured into several steps:
 
+### Evaluation Script
```

Comment:
you mention the math problem training, but where is the notebook for it?

Reply:
There is no notebook. The line after mentions the `eval.py` script